integreat_cms/cms/migrations/0078_alter_poi_category.py

```diff
@@ -0,0 +1,44 @@
+# Generated by Django 3.2.20 on 2023-07-25 15:46
+
+from django.db import migrations, models
+
+from ..constants import poicategory
+
+
+# pylint: disable=unused-argument
+def check_and_update_poicategory(apps, schema_editor):
+    """
+    This function checks whether the default POI category "Other" exists and cretes it if not.
+    It checks furthermore whether each location has a category and assigns the default category if not.
+    """
+    POICategory = apps.get_model("cms", "POICategory")
+    POI = apps.get_model("cms", "POI")
+
+    default_category, _ = POICategory.objects.get_or_create(
+        icon=poicategory.OTHER, color=poicategory.DARK_BLUE
```

Comment:
```suggestion
        icon=poicategory.OTHER, defaults={"color": poicategory.DARK_BLUE},
```
The color is not so important I'd say, so I think it's enough to only set this when the other category needs to be created.
One thought I just had: It's possible to create multiple "other" categories, right? Then, this would throw an error :neutral_face: 
Maybe my solution wasn't such idea after all? :see_no_evil: 
Also, we might have to create at least one translation?

Reply:
Right...`get_or_create` throws an error if there are more than one category with `icon=other` 🤦 I would query category objects for `icon=other` and take the first one if any exists and create a new one else, because this query will be anyway necessary if `get_or_create` throws the error.

To create a translation for the newly generated other category (if any does not exists), there is a difficulty: no language exists at the moment of reading this migration but `POITranslation` model needs one......